lib/AST/Decl.cpp

```diff
@@ -1390,8 +1390,35 @@ unsigned PatternBindingDecl::getPatternEntryIndexForVarDecl(const VarDecl *VD) c
   return ~0U;
 }
 
+Expr *PatternBindingEntry::getOrigInitAsWritten() const {
+  // Before the addition of property wrappers, the following would work:
+  // return InitContextAndIsText.getInt() ? nullptr : InitExpr.Node;
+  // Consider: @Blah(17) var foo: Foo = 18
+  // The code above will return:
+  // Clamped(wrappedValue: 17, min: 0, max: 255)
+  // but what we want is the "18".
```

Comment:
The "before the addition of property wrappers" bit doesn't really belong in the comment; that's more like a commit message, which would show the old code in the diff anyway. Can you settle on one example (`Clamped` is nice because it's in SE-0258) and put the comment about "we want the 18" down by the call to `findOriginalPropertyWrapperInitialValue`

Reply:
I think it's gone.